policies/policy_service.go

```diff
@@ -126,3 +112,92 @@ func (s policiesService) ViewPolicyByID(ctx context.Context, token string, polic
 	}
 	return res, nil
 }
+
+func (s policiesService) EditPolicy(ctx context.Context, token string, pol Policy, format string, policyData string) (Policy, error) {
+	ownerID, err := s.identify(token)
+	if err != nil {
+		return Policy{}, err
+	}
+
+	// Used to get the policy backend and validate it
+	plcy, err := s.repo.RetrievePolicyByID(ctx, pol.ID, ownerID)
+	if err != nil {
+		return Policy{}, err
+	}
+	pol.Backend = plcy.Backend
+	pol.MFOwnerID = ownerID
+
+	err = validatePolicyBackend(&pol, format, policyData)
+	if err != nil {
+		return Policy{}, err
+	}
+
+	err = s.repo.UpdatePolicy(ctx, ownerID, pol)
```

Comment:
Before saving to the repo we want to increase the `pol.Version` by 1

Reply:
Done